webui/react/src/pages/FlatRuns/FlatRuns.tsx

```diff
@@ -0,0 +1,759 @@
+            newSettings.exclusions = Array.from(excludedSet);
+          } else {
+            const includedSet = new Set(newSettings.selections);
+            rowRangeToIds(range).forEach((id) => includedSet.delete(id));
+            newSettings.selections = Array.from(includedSet);
+          }
+
+          break;
+        case 'remove-all':
+          newSettings = DEFAULT_SELECTION;
+
+          break;
+        case 'set':
+          if (!range) return;
+          newSettings = {
+            ...DEFAULT_SELECTION,
+            selections: Array.from(rowRangeToIds(range)),
+          };
+
+          break;
+      }
+
+      updateSettings({ selection: newSettings });
+    },
+    [rowRangeToIds, settings.selection, updateSettings],
+  );
+
+  const handleContextMenuComplete: ContextMenuCompleteHandlerProps<ExperimentAction, FlatRun> =
+    useCallback(() => {}, []);
+
+  const handleColumnsOrderChange = useCallback(
+    (newColumnsOrder: string[]) => {
+      updateSettings({ columns: newColumnsOrder });
+    },
+    [updateSettings],
+  );
+
+  const handleSortChange = useCallback(
+    (sorts: Sort[]) => {
+      setSorts(sorts);
+      const newSortString = makeSortString(sorts.filter(validSort.is));
+      if (newSortString !== sortString) {
+        resetPagination();
+      }
+      updateSettings({ sortString: newSortString });
+    },
+    [resetPagination, sortString, updateSettings],
+  );
+
+  const getRowAccentColor = (rowData: FlatRun) => {
+    return colorMap[rowData.id];
+  };
+
+  const handlePinnedColumnsCountChange = useCallback(
+    (newCount: number) => updateSettings({ pinnedColumnsCount: newCount }),
+    [updateSettings],
+  );
+
+  const getHeaderMenuItems = useCallback(
+    (columnId: string, colIdx: number): MenuItem[] => {
+      if (columnId === MULTISELECT) {
+        const items: MenuItem[] = [
+          selection.rows.length > 0
+            ? {
+                key: 'select-none',
+                label: 'Clear selected',
+                onClick: () => {
+                  handleSelectionChange?.('remove-all');
+                },
+              }
+            : null,
+          ...[5, 10, 25].map((n) => ({
+            key: `select-${n}`,
+            label: `Select first ${n}`,
+            onClick: () => {
+              handleSelectionChange?.('set', [0, n]);
+              dataGridRef.current?.scrollToTop();
+            },
+          })),
+          {
+            key: 'select-all',
+            label: 'Select all',
+            onClick: () => {
+              handleSelectionChange?.('add-all');
+            },
+          },
+        ];
+        return items;
+      }
+      const column = Loadable.getOrElse([], projectColumns).find((c) => c.column === columnId);
+      if (!column) {
+        return [];
+      }
+
+      const filterCount = formStore.getFieldCount(column.column).get();
+
+      const BANNED_FILTER_COLUMNS = ['searcherMetricsVal', 'searcherType'];
```

Comment:
double-checking it's intentional to include `searcherType` here because it's not on `F_ExperimentList`